docs/source/trainer.mdx

```diff
@@ -20,6 +20,10 @@ We also support a `RewardTrainer` that can be used to train a reward model.
 
 [[autodoc]] SFTTrainer
 
+## DPOTrainer
```

Comment:
What do you think about adding one line in `index.mdx` linking to the doc page of `DPOTrainer` similarly as `SFTTrainer`, `RewardTrainer`, etc. ?